lib/enum.js

```diff
@@ -1,61 +1,51 @@
 'use strict';
 
-class Enum {}
+const NaE = Symbol('NotAnEnum');
 
-const enumArray = values => class extends Enum {
-  constructor(val) {
-    super();
-    this.value = this.constructor.key(val);
+class Enum {
+  static get NaE() {
```

Comment:
Then it will be possible to change its value via simple assignment `Enum.NaE = 123`. With the current solution, you will have to `delete` it and only then you will be able to change it. Though I can change it if you want (or use Object define non-configurable so that it will be impossible to change this at all).

Reply:
We don't need to protect code constructions against idiots, it's javascript anyone can `delete Function.prototype.call`

Reply:
We can follow `Number.NaN` approach, so `Object.getOwnPropertyDescriptor(Number, 'NaN')`
returns: `{ value: NaN, writable: false, enumerable: false, configurable: false }`

Reply:
Yeah, that's what I suggested at the end there. Ok, I'll do that.